plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryNamedRelationHandle.java

```diff
@@ -78,6 +81,12 @@ public Optional<String> getComment()
         return comment;
     }
 
+    @JsonProperty
+    public Optional<String> getConnectionId()
```

Comment:
nit: The String value is unused actually. boolean is sufficient.

Reply:
Fixed